docs/sources/tempo/operations/schema.md

```diff
@@ -232,12 +258,12 @@ For speed and ease-of-use, we are projecting several values to columns at the tr
 * Root service/span names/StartTimeUnixNano - These are selected properties of the root span in each trace (if there is one). These are used for displaying results in the Grafana UI. These properties are computed at ingest time and stored once for efficiency, so we don't have to find the root span.
 * `DurationNanos` - The total trace duration, computed at ingest time. This powers the min/max duration filtering in the current Tempo search and is more efficient than scanning the spans duration column. However, it may go away with TraceQL or we could decide to change it to span-level duration filtering too.
 
-## Dedicated columns
+## Well-known attributes
```

Comment:
The changes to the two names helps clarify and differentiate these capabilities. Thank you for making these changes.